### PR TITLE
Parser of dates

### PR DESCRIPTION
**Pull request by howareyouman**
Added new class

### CODE REVIEW

File: src/main/java/com/w6/nlp/DateTimeParser.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+ * Class to parse date.
+ */
+
+public class DateTimeParser {
+    /**
+     * String date parser.
+     * @param text    This is the text to parse.
+     */
+
+    public static List<String> parseDateAndDateFromString(String text) throws Exception {
```
Comment (towelenee) [inline src/main/java/com/w6/nlp/DateTimeParser.java]: And notNull notation or just leave it as it is. Exception is too wide. In general it's better to use more specific exceptions.

File: src/main/java/com/w6/nlp/DateTimeParser.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
+/**
+ * Created by arseny on 25.02.16.
+ */
+import edu.stanford.nlp.simple.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Class to parse date.
```
Comment (towelenee) [inline src/main/java/com/w6/nlp/DateTimeParser.java]: useless coment

File: src/main/java/com/w6/nlp/DateTimeParser.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
+/**
+ * Created by arseny on 25.02.16.
```
Comment (towelenee) [inline src/main/java/com/w6/nlp/DateTimeParser.java]: We use git annotate. Copywrite can be confusing

**Comment by towelenee** [comment]
Tests and integrate it to a site.

File: src/test/java/DateTest.java [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class DateTest {
+    @Test
+    public void testDate0() {
```
Comment (towelenee) [inline src/test/java/DateTest.java]: Meaningful names to test

File: src/test/java/DateTest.java [hunk 5]
```diff
@@ -0,0 +1,33 @@
+import com.w6.nlp.DateTimeParser;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class DateTest {
```
Comment (towelenee) [inline src/test/java/DateTest.java:9]: Can we add test with real dates like "first july" "1st july", "1st july of 2015" and so on?

File: src/main/java/com/w6/nlp/DateTimeParser.java [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.validation.constraints.NotNull;
+
+public class DateTimeParser {
+
+    @NotNull
+    public static List<String> parseDateAndDateFromString(String text) {
```
Comment (towelenee) [inline src/main/java/com/w6/nlp/DateTimeParser.java]: ParseDateAndTime
FromString is obvious from parameters.
And not @notNull comment was about parameter.
@notNull String text

File: src/test/java/DateTest.java [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+    @Test
+    public void testDate0() {
+        List<String> res = DateTimeParser.parseDateAndDateFromString("I wend to school yesterday.");
```
Comment (towelenee) [inline src/test/java/DateTest.java]: Wend -> went. Typo

File: src/test/java/DateTest.java (hunk 5 above)
Comment (howareyouman) [inline src/test/java/DateTest.java:9]: Yes, we can. I've added some.

File: src/main/java/com/w6/nlp/DateTimeParser.java [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.validation.constraints.NotNull;
+
+public class DateTimeParser {
+
+    
+    public static List<String> parseDateAndDateFromString(@NotNull String text) {
```
Comment (towelenee) [inline src/main/java/com/w6/nlp/DateTimeParser.java]: DateAndDate -> DateAndTime

File: src/test/java/DateTest.java [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+        List<String> howItShouldBe = new ArrayList<String>();
+        howItShouldBe.add("tomorrow");
+        assertTrue(res.equals(howItShouldBe));
+    }
+    
+    @Test
+    public void testComplexDate() {
+        List<String> res = DateTimeParser.parseDateAndDateFromString("I will come back on 1st july of 2016.");
+        List<String> howItShouldBe = new ArrayList<String>();
+        howItShouldBe.add("1st");
```
Comment (towelenee) [inline src/test/java/DateTest.java:30]: Indent

**Pull request merged**